views/profile_transactions.ejs

```diff
@@ -0,0 +1,133 @@
+    <link rel='stylesheet' href='/stylesheets/profile.css' />
+
+
+    <!-- Navbar Stuff -->
+    <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #845EC2;">
+
+      <a class="navbar-brand text-white" href="../"> <p class="fas fa-home"></p>  </a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <a class="navbar-brand text-white" href="../profile"> <p class="fas fa-user"></p>  </a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse text-white" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+
+          <li class="nav-item dropdown text-white">
+            <a class="nav-link dropdown-toggle text-white" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Browse
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+              <a class="dropdown-item" href="#">Pick-up</a>
+              <a class="dropdown-item" href="#">Deliver</a>
+              <a class="dropdown-item" href="#">Service</a>
+              <a class="dropdown-item" href="#">Intangible</a>
+            </div>
+          </li>
+
+        </ul>
+
+        <form class="form-inline my-2 my-lg-0">
+          <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+          <button class="btn btn-outline-success my-2 my-sm-0" type="submit"><i class="fas fa-search"></i></button>
+        </form>
+      </div>
+    </nav>
+
+
+    <!-- Profile Page -->
+    <div class="row profile">
+      <div class="col-md-3">
+        <div class="profile-sidebar">
+          <!-- SIDEBAR USERPIC -->
+          <div class="profile-userpic">
+            <img src="../public/images/default_profile.png" alt="">
+          </div>
+          <!-- SIDEBAR USER TITLE -->
+          <div class="profile-usertitle">
+            <div class="profile-usertitle-name"><%= user.username %></div>
+            <div class="profile-usertitle-job"><%= user.name %></div>
+          </div>
+          <!-- SIDEBAR MENU -->
+          <div class="profile-usermenu">
+            <ul class="nav">
+              <li>
+                <a href="/profile">
+                <i class="glyphicon glyphicon-home"></i>My items
+                </a>
+              </li>
+              <li class="active">
+                <a href="#">
+                <i class="glyphicon glyphicon-user"></i>My transaction history
```

Comment:
Er.. we were using font awesome, not glyphicon haha - it's okay if they look the same though - check the user button on the navbar at the top for reference.